flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterCatalogComment.java

```diff
@@ -31,27 +34,26 @@
 /** ALTER CATALOG catalog_name COMMENT 'comment'. */
 public class SqlAlterCatalogComment extends SqlAlterCatalog {
 
-    private final SqlNode comment;
+    private final SqlCharStringLiteral comment;
 
     public SqlAlterCatalogComment(
-            SqlParserPos position, SqlIdentifier catalogName, SqlNode comment) {
+            SqlParserPos position, SqlIdentifier catalogName, SqlCharStringLiteral comment) {
         super(position, catalogName);
         this.comment = requireNonNull(comment, "comment cannot be null");
     }
 
     @Override
     public List<SqlNode> getOperandList() {
-        return ImmutableNullableList.of(catalogName, comment);
+        return ImmutableNullableList.of(getName(), comment);
     }
 
-    public SqlNode getComment() {
+    public SqlCharStringLiteral getComment() {
```

Comment:
Can't we just return string already? No outside consumer really needs `SqlCharStringLiteral` anymore right?